src/test/java/codeu/controller/AdminServletTest.java

```diff
@@ -0,0 +1,67 @@
+package codeu.controller;
+
+import static org.junit.Assert.*;
```

Comment:
Avoid wildcard imports. Please explicitly import what you are using:

https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports